### PR TITLE
Feature  transaction details modal

### PR DESCRIPTION
**Pull request by jackholt44**
Adding transaction detail view - pulling in Item, Account and Transaction details.

At current the list item is a SFC - need to make react component to pass the transaction ID through as prop  - so at current you have to click the Item Name to get the detail view

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,14 +0,0 @@
-SERVER_PORT=8130
```
Comment (atreidesend) [inline .env.example]: why was this deleted?

Comment (jackholt44) [reply .env.example]: Deleted in error

Comment (jackholt44) [reply .env.example]: same as other PR - assume no point re-adding

File: client/actions/TransactionActions.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const closeCategorizeTransactionModal = createAction(
 
 export const categorizeTransactionParentIdSelected = createAction<
   number | null
->('categorize_TRANSACTION_PARENT_ID_SELECTED')
+>('CATEGORISE_TRANSACTION_PARENT_ID_SELECTED')
```
Comment (atreidesend) [inline client/actions/TransactionActions.ts]: z
